tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Equipment Loader
+
+## Overview
+ABEL is a tool allowing systems integrators to easily and efficiently create [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md) and [UDMI Site Models](https://github.com/faucetsdn/udmi/blob/master/docs/specs/site_model.md).
+
+Previously, systems integrators had to a create a building configuration file by hand, but since building config is designed to be a machine=readable document, it is not human-friendly. The ABEL spreadsheet provides a frontend interface for gathering the informtion required to create a Building Config or a Site Model. ABEL also allows users to edit a Building Config file or UDMI Site Model by loading  either document into the frontend spreasheet and exporting it back out.
+
+#### UDMI
+ABEL does not yet support UDMI Site Model, but is planned to in the coming months.
+
+## Setup and installation
+Similar to the [GUID Generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator), ABEL's setup process depends on the instance validator. It is best to run the global setup script to setup the entire tools library which ABEL depends on
+
+1. Run: `git clone https://github.com/google/digitalbuildings.git`
```

Comment:
This is one way of doing it, I suggest to omit 1 and 2?

Reply:
I see a lot of github docs that contain the entire installation process starting with `git clone` so I'd like to keep these two unless it really is that unsightly.